src/cls/IPM/Utils/Update/Controller.cls

```diff
@@ -2,55 +2,38 @@ Class %IPM.Utils.Update.Controller
 {
 
 ClassMethod ApplyUpdatesInOrder(
-	moduleName As %String,
-	toVersion As %String,
+	module As %IPM.Storage.Module,
+	fromVersion As %String,
+	logFile As %Library.File = "",
 	verbose As %Boolean = 0)
 {
-    ///////////////////////////////////////////////////////////////////////////////////////////
-    ////////////////////////// Hard-Coding Values During Development //////////////////////////
-    ///////////////////////////////////////////////////////////////////////////////////////////
-    //set moduleName = "update-test"
-    //set toVersion = "3"  // say current version is 1
-    set tModule = ##class(%IPM.Storage.Module).NameOpen(moduleName,,.tSC)
-    ///////////////////////////////////////////////////////////////////////////////////////////
-
-    if verbose { w !, "Attempting to acquire lock on ^IPM("_moduleName_")" }
-    lock +^IPM(moduleName):3
+    if verbose { w !, "Acquiring lock on ^IPM("_module.Name_")" }
+    lock +^IPM(module.Name):3
```

Comment:
Use a longer more specific global name for locking + have a macro for it to avoid typos (no direct global refs is the general rule). You can also use the LockManager class to simplify locking if you want.